### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by nouman17610**
I implemented the following

- Contact form
- Form checks
- Form styling
- Professional Readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -268,7 +268,29 @@ <h3>Tools</h3>
...
              </ul> 
            </div>
-     </section>
+     <div class="contact-form" id="contact_me">
+      <div class="form-area">
+      <div class="contact-heading">
+        <h3>Contact me</h3>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+      </div>
+      <form action="https://formspree.io/f/{form_id}" onsubmit="return validateForm()" method="post" required>
```
Comment (muneebulrehman) [inline index.html]: When I try to submit the form, I get this error. Kindly fix this issue. Kindly add a proper link to the `action` attribute.
image

File: index.html [hunk 2]
```diff
@@ -268,7 +268,29 @@ <h3>Tools</h3>
                </li>
              </ul> 
            </div>
-     </section>
+     <div class="contact-form" id="contact_me">
```
Comment (muneebulrehman) [inline index.html:271]: The **Contact** section should span through the entire width on the mobile screen. However, in your case, there seems to be some sort of padding on this section. Kindly remove it. The main purpose of this module is to replicate the pixel-perfect design of the figma sheets. Kindly fix this issue.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -268,7 +268,29 @@ <h3>Tools</h3>
...
+      </div>
+      <form action="https://formspree.io/f/{form_id}" onsubmit="return validateForm()" method="post" required>
+        <div>
+          <input type="text" id="name" name="name" placeholder="Enter your name" size="40" max-length="30" required>
+        </div>
+        <div>
+          <input type="email" id="email" name="email" placeholder="Enter your email" size="40" required>
+        </div>
+        <div>
+          <textarea name="message" columns="30" rows="10" placeholder="Enter your message here" required></textarea>
```
Comment (muneebulrehman) [inline index.html]: `textarea` should also have a `maxlength` attribute equal to 500. Kindly add the same.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @nouman17610 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job adding the form. 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — APPROVED** [review]
Hi @nouman17610  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**